src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -720,6 +731,7 @@ private void MakeLayout(int width, int maxLines = int.MaxValue)
 					.SetEllipsizedWidth(width)
 					.SetAlignment(_layoutAlignment)
 					.SetIncludePad(true)
+					.SetJustificationMode(_justificationMode)
```

Comment:
`JustificationMode` was introduced in Android 8, this will likely fail at runtime on devices running older versions. 

You need to check the Android runtime version, like this:

https://github.com/unoplatform/uno/blob/e12cbcb00a55ee04b145141728f37e95a318951f/src/Uno.UWP/Phone/Devices/Notification/VibrationDevice.Android.cs#L44-L47

Reply:
Nice catch!